src/main/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderController.java

```diff
@@ -54,7 +51,7 @@
         })
     @GetMapping(path="offender/{crn}/convictions/{convictionId}/requirements", produces = APPLICATION_JSON_VALUE)
     public @ResponseBody
-    Mono<List<Requirement>> getRequirements(@PathVariable String crn, @PathVariable String convictionId) {
-        return offenderService.getConvictionRequirements(crn, convictionId);
+    RequirementsResponse getRequirements(@PathVariable String crn, @PathVariable String convictionId) {
```

Comment:
@katie-cawthorne I had to change this to an Object rather than returning a list directly as Swagger wasn't able to handle the Mono<List>, hope that's ok. Returning a flat list can cause problems for some clients as well so if possible it's better to just return an object with a single array field.